src/Symfony/Component/HttpKernel/DependencyInjection/AddClassesToCachePass.php

```diff
@@ -39,12 +39,27 @@ public function process(ContainerBuilder $container)
         $classes = array();
```

Comment:
Ideally I'd just append everything to 1 big array and then do `array_merge([], ...$classes)`, but 5.5 doesn't support that